lib/reducers/call-taker.js

```diff
@@ -7,6 +7,7 @@ import {getTimestamp} from '../util/state'
 const { randId } = coreUtils.storage
 
 const UPPER_RIGHT_CORNER = {x: 604, y: 53}
+const LOWER_RIGHT_CORNER = {x: 504, y: 400}
```

Comment:
Can you post a screenshot of what you're seeing?

Reply:
The window is appearing in the bottom right. Seems like it could be more towards the center.

<img width="1337" alt="Screen Shot 2021-01-15 at 4 51 09 PM" src="https://user-images.githubusercontent.com/3112493/104792062-f63b9700-5751-11eb-8390-c6e6d03b90e6.png">